book/src/concepts/what-is-a-transaction.md

```diff
@@ -29,6 +29,27 @@ Transactions consist of:
 - a gas object - the `Coin` object used to pay for the transaction;
 - gas price and budget - the cost of the transaction;
 
+## Inputs
+
+Transaction inputs are the arguments for the transaction and are split between 2 types:
+- Pure arguments: These are mostly [primitive types](../move-basics/primitive-types.html) with some
+extra additions. A pure argument can be:
+    - boolean
+    - integer (u8, u16, u32, u64, u128, u256)
+    - address
+    - std::ascii::String
+    - std::string::String
+    - vector of pure types
+    - Option of pure types
+    - ID, a special type that refers to a unique identifier of an object
```

Comment:
Ditto, not a special type, defined in `sui::object::ID`.

Reply:
object/object-model seems more appropriate as it refers to the UID.

Reply:
After all I linked `storage/uid-and-id`. I also referred to `object/object-model` as "See also".
[a49b910](https://github.com/MystenLabs/move-book/pull/91/commits/a49b910dc4a8537d8bacbc1b989f34b7c5004ab3)